### ISSUE TITLE
fn start gives two mount warnings

### ISSUE THREAD

**Issue by carimura** [fnproject/fn#219]
```
❯ fn start
mount: permission denied (are you root?)
Could not mount /sys/kernel/security.
AppArmor detection and --privileged mode might break.
mount: permission denied (are you root?)

time="2017-08-14T18:11:34Z" level=info msg="datastore dialed" datastore=sqlite3 max_idle_connections=256

      ______
     / ____/___
    / /_  / __ \
   / __/ / / / /
  /_/   /_/ /_/

time="2017-08-14T18:11:34Z" level=info msg="Serving Functions API on address `:8080`"
```

Things still work, but the two (duplicate) warnings are jarring...

**Comment by mattstep** [comment]
@carimura Can you clarify which docker you tested this with on mac? Using the current docker on mac (with xhyve), I don't get these errors. I'm curious if this was fixed or if you're running boot2docker or something like that. I see:

    mattstep-mba:fn mattstep$ ~/go/bin/fn start
    INFO[0000] datastore dialed                              datastore=sqlite3 max_idle_connections=256
    INFO[0000] no docker auths from config files found (this is fine)  error="open 
/Users/mattstep/.dockercfg: no such file or directory"
    INFO[0000] available memory                              ram=2147483648
    
            ______
           / ____/___
          / /_  / __ \
         / __/ / / / /
        /_/   /_/ /_/
            v0.3.153
    
    INFO[0000] Serving Functions API on address `:8080`

**Comment by carimura** [comment]
hmm... @mattstep 

```
/tmp
❯ docker --version
Docker version 17.09.0-ce, build afdb6d4

/tmp
❯ fn start
mount: permission denied (are you root?)
Could not mount /sys/kernel/security.
AppArmor detection and --privileged mode might break.
mount: permission denied (are you root?)
time="2017-10-13T05:45:02Z" level=info msg="datastore dialed" datastore=sqlite3 max_idle_connections=256
time="2017-10-13T05:45:02Z" level=info msg="no docker auths from config files found (this is fine)" error="open /root/.dockercfg: no such file or directory"
time="2017-10-13T05:45:02Z" level=info msg="available memory" ram=1601916928

        ______
       / ____/___
      / /_  / __ \
     / __/ / / / /
    /_/   /_/ /_/
        v0.3.153

time="2017-10-13T05:45:02Z" level=info msg="Serving Functions API on address `:8080`"
```

**Comment by treeder** [comment]
@mattstep looks like you're running the binary directly, not the docker container.

**Comment by mattstep** [comment]
@treeder That makes a difference. Alright, I can reproduce this now.

**Comment by skinowski** [comment]
because dind image is run without --privileged flag, so mounting /sys/kernel/security fails?

cli start.go still refers to funcy/functions (should that be fnproject/functions?), which possibly has a script somewhere like below?

https://github.com/moby/moby/blob/master/hack/dind

**Comment by denismakogon** [comment]
No longer the issue on Mac with Fn CLI version 0.4.30:
```
fn start
time="2017-12-18T11:09:37Z" level=info msg="started tracer" url=
time="2017-12-18T11:09:37Z" level=info msg="datastore dialed" datastore=sqlite3 max_idle_connections=256
time="2017-12-18T11:09:37Z" level=info msg="no docker auths from config files found (this is fine)" error="open /root/.dockercfg: no such file or directory"
time="2017-12-18T11:09:37Z" level=info msg="available memory" availMemory=10844198093 cgroupLimit=9223372036854771712 headRoom=1204910899 totalMemory=12049108992
time="2017-12-18T11:09:37Z" level=info msg="sync and async reservations" ramAsync=8675358475 ramAsyncHWMark=6940286780 ramSync=2168839618
time="2017-12-18T11:09:37Z" level=info msg="Fn serving on `:8080`" type=full

        ______
       / ____/___
      / /_  / __ \
     / __/ / / / /
    /_/   /_/ /_/
        v0.3.243

```

**Comment by carimura** [comment]
confirmed, those errors are gone. there are some new Docker ones that I think someone pointed out:

```
❯ fn start
can't create unix socket /var/run/docker.sock: device or resource busy
time="2017-12-18T16:35:50Z" level=info msg="started tracer" url=
time="2017-12-18T16:35:50Z" level=info msg="datastore dialed" datastore=sqlite3 max_idle_connections=256
time="2017-12-18T16:35:50Z" level=info msg="no docker auths from config files found (this is fine)" error="open /root/.dockercfg: no such file or directory"
time="2017-12-18T16:35:50Z" level=info msg="available memory" availMemory=1509670912 cgroupLimit=9223372036854771712 headRoom=268435456 totalMemory=1778106368
time="2017-12-18T16:35:50Z" level=info msg="sync and async reservations" ramAsync=1207736730 ramAsyncHWMark=966189384 ramSync=301934182

        ______
       / ____/___
      / /_  / __ \
     / __/ / / / /
    /_/   /_/ /_/
        v0.3.243

time="2017-12-18T16:35:50Z" level=info msg="Fn serving on `:8080`" type=full
```

**Comment by rdallman** [comment]
there are no more warnings, only info log lines, this issue is closed
